test/time-ago.js

```diff
@@ -37,6 +37,20 @@ suite('time-ago', function() {
     assert.equal(root.children[0].textContent, 'now')
   })
 
+  test('rewrites from now past datetime to months ago', function() {
+    const now = new Date(Date.now() - 3 * 30 * 24 * 60 * 60 * 1000).toISOString()
+    const time = document.createElement('time-ago')
+    time.setAttribute('datetime', now)
+    assert.equal(time.textContent, '3 months ago')
+  })
+
+  test('rewrites from now past datetime to years ago', function() {
+    const now = new Date(Date.now() - 12 * 30 * 24 * 60 * 60 * 1000).toISOString()
+    const time = document.createElement('time-ago')
+    time.setAttribute('datetime', now)
+    assert.equal(time.textContent, 'last year')
```

Comment:
What does "last year" mean if it's January right now? Can we instead have it say something like “1 year ago”?

Reply:
It means "in the last 12 months" I believe.

The `last year` text comes from us setting the `numeric` option to `auto` and not the default `always`. It's the option that gives us back timestamps such as `yesterday`.

- [RelativeTimeFormat - Parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RelativeTimeFormat#Parameters)

We can special case the time there to set the `numeric` option to `always`.